test/controllers/api/rhsm/candlepin_proxies_controller_test.rb

```diff
@@ -50,7 +50,7 @@ module Katello
         Resources::Candlepin::Consumer.stubs(:get)
 
         ::Katello::Host::SubscriptionFacet.expects(:new_host_from_facts).returns(@host)
-        assert_sync_task(::Actions::Katello::Host::Register, @host, {'facts' => @facts}, nil, [@activation_key])
+        ::Katello::RegistrationManager.expects(:register_host).with(@host, {'facts' => @facts}, nil, [@activation_key])
```

Comment:
we need to make sure we have a ping check on both of these to ensure that all the services are functional